packages/botbuilder-filter-middleware/src/filterMiddleware.ts

```diff
@@ -0,0 +1,26 @@
+import { Activity, Middleware, ConversationResourceResponse, MiddlewareSet } from 'botbuilder';
```

Comment:
Instead of creating a separate package want to just add it to botbuilder-toybox-middleware?  Otherwise I'd say just publish it out of your own repo